docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    gdf = gpd.GeoDataFrame(df, geometry=geometry, crs="EPSG:4326")
</pre><p>we can get rid of the zip in the line above if you want to do something like</p>

<pre class="ql-syntax" spellcheck="false">gdf = gpd.GeoDataFrame(df, geometry=gpd.points_from_xy(df['longitude'], df['latitude']), crs="EPSG:4326")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>